crates/red_knot_python_semantic/src/types.rs

```diff
@@ -402,9 +399,16 @@ fn symbol_from_bindings<'db>(
 /// If we look up the declared type of `variable` in the scope of class `C`, we will get
 /// the type `int`, a "declaredness" of [`Boundness::PossiblyUnbound`], and the information
 /// that this comes with a [`TypeQualifiers::CLASS_VAR`] type qualifier.
+#[derive(Debug)]
 pub(crate) struct SymbolAndQualifiers<'db>(Symbol<'db>, TypeQualifiers);
 
 impl SymbolAndQualifiers<'_> {
+    /// Smart constructor to create a [`SymbolAndQualifiers`] instance with a [`TodoType`] type
```

Comment:
Minor naming nit: I think usually a "smart constructor" is understood to apply some additional consistency checks or simplifications. I wouldn't call this a smart constructor, just... a constructor.